ebpf/network-tc-filter/injection.bpf.c

```diff
@@ -134,10 +161,11 @@ int cls_classifier_paths(struct __sk_buff *skb)
         p[i] = load_byte(skb, skb_info.data_off + i);
     }
 
-    char path[MAX_PATH_LEN];
+    char path[MAX_PATH_LEN] = {0};
```

Comment:
what does this do? Seems a bit weird to me, why not initializing to empty string.
Each empty string should end by \0 which I think was your objective here?

```suggestion
    char path[MAX_PATH_LEN] = "";
```

Reply:
Both char path[MAX_PATH_LEN] = {0}; and char path[MAX_PATH_LEN] = ""; achieve the same result of initializing the array with null characters (\0). However, char path[MAX_PATH_LEN] = {0}; is more explicit and commonly used in C programming to ensure all elements are set to 0.  

Here's a brief comparison:  
char path[MAX_PATH_LEN] = {0};: Explicitly initializes all elements of the array to 0.
char path[MAX_PATH_LEN] = "";: Initializes the first element to 0 and relies on the compiler to zero-initialize the rest.

Using char path[MAX_PATH_LEN] = {0}; is generally preferred for clarity and explicitness.

Reply:
gotcha, i disagree with copilot to me it's esotheric lol. Can you add a comment?

Reply:
I have no preference, I can use `""` if you prefere 👍

Reply:
🤷‍♀️ I have no preference with this explanation, it seems like it's a better practice to {0} so let's go with that.

Reply:
After all, I prefer to use a simple `""`. I think it make more sense to use `{0}` when you are initialising a list.